requirements.txt

```diff
@@ -8,7 +8,7 @@ rich
 coverage
 pytest
 isort
-black==23.3.0
```

Comment:
please revert this change back to 23.3.0

Reply:
@kingxleyy i didnt ask you to delete the file i asked to simply revert the change in `requirements.txt` i **dont** want you to update the black version in requirements.txt. Please try again or simply open another PR.